File: Other/2025/67. Add Binary.js
/**
 * @param {string} a
 * @param {string} b
 * @return {string}
 */
var addBinary = function (a, b) {
  return (BigInt(`0b${a}`) + BigInt(`0b${b}`)).toString(2);
};

// Test cases
console.log(addBinary("11", "1"));

File: Other/2025/67. Add Binary.py
class Solution:
    def addBinary(self, a: str, b: str) -> str:
        return bin(int(a, 2) + int(b, 2))[2:]
    
# Test cases
print(Solution().addBinary("11", "1")) # 100
print(Solution().addBinary("1010", "1011")) # 10101